software/fasten-health.yml

```
@@ -1,6 +1,6 @@
 name: Fasten Health
 website_url: https://github.com/fastenhealth/fasten-onprem/
-description: Fasten is an open-source, self-hosted, personal/family electronic medical record aggregator, designed to integrate with 100,000's of insurances/hospitals/clinics in the United States.
+description: Personal/family electronic medical record aggregator, designed to integrate with 100,000's of insurances/hospitals/clinics in the United States.
```

Is the number really necessary here? It looks a bit awkward.
```suggestion
description: Personal/family electronic medical record aggregator, designed to integrate with hundreds of thousands of insurances/hospitals/clinics in the United States.
```